### ISSUE TITLE
It would be better if we group rules into categories

### ISSUE THREAD

**Issue by zoonooz** [ambas/RealManRules#7]
As the list is growing bigger and bigger, should we create separate categories for rules? for example
- Work
- Life
- etc

**Comment by kunpakon** [comment]
I think it not much interesting
You must prefix category name with "Real Man with ..."
for example 

# Real Man with mustache
- Real man with mustache must not use gillette mach3 with aloe vera
- Real man with mustache must ride a buffalo

**Comment by ambas** [comment]
Good idea!, I agree with you guys that this repo is getting bigger and bigger and categorize will make it easier to browse. I will try to craft a prototype and we can discuss together.

**Comment by kunpakon** [comment]
Nice, Can we meet?
I know the great board game cafe at Silom, named Get Board Game on Decho road
It's a cool place man!
